### DISCUSSION TITLE
supermerill out of action

### DISCUSSION THREAD

**Discussion by darco415** [supermerill/SuperSlicer#4193] [General]
Merill stated on discord (3/2) that he "broke" his knee and is currently recuperating, hence the pause here on development. Get well soon Merrill!

**Comment by lessbones** [comment]
who needs a knee for programming?


(kidding, kidding.)

**Comment by neophyl** [comment]
Thanks for letting us know.

**Comment by foreachthing** [comment]
@supermerill: Je te souhaite un bon rétablissement. Meilleures salutations.
